city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,196 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+
+
+class CookJusticeAdvisorySpider(CityScrapersSpider):
+    name = "cook_justice_advisory"
+    agency = "Cook County Justice Advisory"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+
+    def __init__(self, *args, **kwargs):
+        self.agenda_map = defaultdict(list)
+        super().__init__(*args, **kwargs)
+
+    def start_requests(self):
+        url = "https://www.cookcountyil.gov/service/justice-advisory-council-meetings"
+        yield scrapy.Request(
```

Comment:
I had missed this before, but to make sure that this is populated before getting to the detail page we'll need to chain it in separate methods. So this could have a new callback `_parse_meetings_page` that looks something like this:

```python
def _parse_meetings_page(self, response):
  self._parse_links(response)
  ... # The rest of it would be everything below in `start_requests`
```

Looking at it again, it could be even simpler to make the piece above the `parse` method, set `start_urls` to `["https://www.cookcountyil.gov/service/justice-advisory-council-meetings"]`, and change what is currently `parse` to something like `_parse_events_page`.

The reason this is important is currently there's the potential for one of the results pages to yield before the links are populated, and by chaining them this way we can be sure that the link map is filled before we try to pull from it